src/Common/Common.Api/Workspaces/WorkspacesClient.cs

```diff
@@ -60,5 +60,34 @@ public object NewWorkspaceAsUser(string workspaceName)
         {
             return (Workspace)this.Client.Groups.CreateGroup(new PowerBI.Api.V2.Models.GroupCreationRequest(name: workspaceName));
         }
+
+        public object MigrateWorkspaceCapacity(Guid workspaceId, Guid capacityId)
+        {
+            if (capacityId != Guid.Empty)
+            {
+                var request = new PowerBI.Api.V2.Models.AssignWorkspacesToCapacityRequest
+                {
+                    CapacityMigrationAssignments = new List<PowerBI.Api.V2.Models.CapacityMigrationAssignment>
```

Comment:
Looking closer at this class, I realize that we don't have a way for the cmdlets to take advantage of the bulk functionality available to us through these lists. Do you intend to add cmdlets later for this?

Reply:
Currently in the spec we only support single workspace migration. Later we might add batch support.